apps/docs/content/guides/platform/read-replicas.mdx

```diff
@@ -97,30 +97,19 @@ A connection pool through Supavisor is also available for each Read Replica. Fin
 
 A load balancer is deployed to automatically balance requests between your Primary database and Read Replicas. Find its endpoint on the [API Settings page](/dashboard/project/_/settings/api).
 
-The load balancer uses a round-robin strategy to route `GET` requests to all available API endpoints, including the Primary database. This means that `GET` requests are randomly and evenly split among databases. Non-`GET` requests can also be sent through this endpoint, and will be routed to the Primary database.
+The load balancer is geo-aware meaning that `GET` requests to this endpoint will automatically be routed to the Read Replica that is closest to your user ensuring the lowest latency. Non-`GET` requests can also be sent through this endpoint, and will be routed to the Primary database.
 
-To call a read-only Postgres function on Read Replicas through the REST API, use the `get: true` [option](/docs/reference/javascript/rpc?queryGroups=example&example=call-a-read-only-postgres-function).
-
-If you remove all Read Replicas from your project, the load balancer and its endpoint are removed as well. Make sure to redirect requests back to your Primary database before removal.
-
-#### Experimental routing
-
-The API load balancer offers an experimental routing mode that builds on top of the existing functionality. It is now possible to use a load balancer endpoint for all Supabase services (Auth, Edge Functions, Realtime, and Storage) meaning there is no need to worry about which endpoint to use in which situations.
+All Supabase services (Auth, Edge Functions, Realtime, and Storage) are supported meaning there is no need to worry about which endpoint to use in which situations.
```

Comment:
@Crispy1975 idk if the rest of the docs here has this but we can add callout that the other services are not yet geo-aware.